data_model/src/block_value.rs

```diff
@@ -24,19 +24,19 @@ declare_item! {
     #[derive(
         Debug, Clone, Display, PartialEq, Eq, Hash, Decode, Encode, Deserialize, Serialize, FfiType, IntoSchema,
     )]
-    #[display(fmt = "Block №{height} (hash: {transactions_hash});")]
+    #[display(fmt = "Block №{height} (hash: {transactions_hash:?});")]
```

Comment:
Given that this is the `Display` implementation, it could be worthwhile to use `unwrap_or("empty")` in this position. In most cases it's `Some`, so we're just making the implementation noisy.

Reply:
Have to fallback to manual implementation of `Display`.

Reply:
Actually i think that we have buggy implementation of `Display`, we would like to print `Hash` of Block not hash of accepted transactions.